src/fizzBuzz.js

```diff
@@ -0,0 +1,17 @@
+  export default function fizzBuzz(){
+
+    var myArray = []
+    for (var i = 1; i <= 100; i++) {
+      if(i % 15 === 0){
+        myArray.push('FizzBuzz')
+      } else if(i % 3 === 0){
+        myArray.push('Fizz')
+      } else if(i % 5 === 0){
+        myArray.push('Buzz')
+      }
+      else {
```

Comment:
Move this line to the end of the curly brace on line `11` for consistency and readability.